simphony/cuds/abc_modeling_engine.py

```diff
@@ -37,16 +37,9 @@ def add_lattice(self, lattice):
 
         Parameters
         ----------
-        lattice: ABCLattice
+        lattice : ABCLattice
             lattice to be added.
 
-        Returns
```

Comment:
add_lattice, add_particle_container and add_mesh in AbcModelingEngine would be like add_lattice, add_particle_container and add_mesh methods in CudsFile.  The methods would add a mesh, lattice or pc to the engine and the method would return a mesh, lattice, or pc which can then be used to update/query the internal representation. 


Comes from https://github.com/simphony/simphony-common/pull/5#issuecomment-55582063:
> 2. return a handle when adding container file